indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/helpers/GenesisHelper.kt

```diff
@@ -0,0 +1,55 @@
+package com.luxoft.blockchainlab.hyperledger.indy.helpers
+
+import java.io.File
+import java.io.FileNotFoundException
+import java.nio.file.Path
+
+
+/**
+ * Utilities for genesis file management
+ */
+object GenesisHelper {
+
+    /**
+     * Get already existing genesis file
+     *
+     * Checks if genesis file exists and returns [File] object.
+     *
+     * @param genesisPath: String - path to genesis file
+     * @return: [File] - genesis file
+     * @throws FileNotFoundException - if genesis file not found
+     */
+    @Throws(FileNotFoundException::class)
+    fun getGenesis(genesisPath: String): File {
+        val genesisFile = File(genesisPath)
+
+        // making this check here for better debugging
+        if (!genesisFile.exists())
+            throw FileNotFoundException("Genesis file not found at $genesisPath")
+
+        return genesisFile
+    }
+
+    /**
+     * Get genesis file if it doesn't exist in system yet
+     *
+     * Creates all required parent directories according to [newFilePath] and rewrites genesis file with [content]
```

Comment:
Method name is misleading, why dont we call it createGenesis?

Reply:
I was thinking it would be nice to have only one method name in each Helper. E.g. getConfig in ConfigHelper, getGenesis in GenesisHelper and so on.

Reply:
A remark for those who think we don't actually need this function - remember Android.

Reply:
If method creates genesis it should be called "create". And again, there maybe logic how to operate with files, similar to IO